src/interpreter/ir_normalisation.jl

```diff
@@ -228,6 +235,44 @@ expressions in the same way as a primitive `:call` expression, i.e. via an `rrul
 """
 splatnew_to_call(x) = Meta.isexpr(x, :splatnew) ? Expr(:call, _splat_new_, x.args...) : x
 
+# XXX: this is probably unsound, but the best workaround so far
+function resolve_unbound_globalrefs(ir, @nospecialize ex)
```

Comment:
What is this doing? If the reference is unbound, should we just error out? This might be obsolete once we set the IR world range to only the interpreter's, maybe?

Reply:
I ran into issues where the `CC.verify_ir` check did not pass because we had unbound `GlobalRef`s in the IR. You should hit the error if you uncomment this pass.